dusty/commands/repos.py

```diff
@@ -28,14 +28,23 @@ def override_repo(repo_name, source_path):
     save_config_value(constants.CONFIG_REPO_OVERRIDES_KEY, config)
     log_to_client('Locally overriding repo {} to use source at {}'.format(repo.remote_path, source_path))
 
-@daemon_command
-def manage_repo(repo_name):
+def _manage_repo(repo_name):
     repo = Repo.resolve(get_all_repos(), repo_name)
     config = get_config_value(constants.CONFIG_REPO_OVERRIDES_KEY)
     if repo.remote_path in config:
         del config[repo.remote_path]
-    save_config_value(constants.CONFIG_REPO_OVERRIDES_KEY, config)
-    log_to_client('Will manage repo {} with Dusty-managed copy of source'.format(repo.remote_path))
+        save_config_value(constants.CONFIG_REPO_OVERRIDES_KEY, config)
+        log_to_client('Will manage repo {} with Dusty-managed copy of source'.format(repo.remote_path))
```

Comment:
This will now print absolutely nothing if the user attempts to manage a repo that's already managed. Doesn't seem good for usability.